6_visualize.yml

```diff
@@ -10,8 +10,22 @@ file_extensions:
   - ind
 
 sources:
+  - 6_visualize/src/create_storm_frame.R
 
 targets:
 
   6_visualize:
     depends:
+
+  # create_storm_frame will soon be called for all frames from within
+  # a task table. just calling it once here to illustrate
+  storm_frame_config:
+    command: viz_config[I(c('bbox','projection','width','height','focus_geoms','secondary_geoms'))]
+  6_visualize/out/example_storm_frame.png:
+    command: create_storm_frame(
+      png_file=target_name, timestep=I('2017-09-26 11:00:00'), config=storm_frame_config,
```

Comment:
I was just thinking about this for the task tables #27 , but how many frames do we want for the gif? I guess this could be stored in `viz_config` as `frame_time_interval`. Any idea what the interval should be for this initial gif? 6 hours?

Reply:
At least for now, how about we have as many frames as there are timesteps in `2_process/out/timesteps.rds`?

Reply:
sounds good

Reply:
I was thinking hourly (frame per hour) as a default, but we'd want the ability to do every 2 hrs, 3 hrs, 6hrs etc as needed

Reply:
Agreed, but I think we could do that subsetting earlier, in `choose_timesteps()`, and then have lighter-weight data to throw around during the visualization phase. Would be a pretty simple addition to viz_config.yml (something like `time_resolution_hrs: 3`) and modification to `choose_timesteps()` (just take every 3rd of the hourly precip timestamps, or get a little fancier with the POSIXct if needed)